.github/workflows/build.yaml

```diff
@@ -76,3 +76,26 @@ jobs:
       with:
         name: system.img
         path: output/system.img
+
+    - name: Package and OTA push
+      if: "contains(env.LAST_COMMIT_MESSAGE, '[release]')"
+      env:
+        AGNOS_UPDATE_URL: https://openpilot.blob.core.windows.net/agnosupdate-ci/${{ github.run_id }}
+        # TODO: switch to comma credentials: https://learn.microsoft.com/en-us/cli/azure/ad/sp?view=azure-cli-latest#az-ad-sp-create-for-rbac-examples
+        AZURE_CREDENTIALS: '{"appId": "fd6e3414-7fee-4bf3-bd85-dd36d3d03bd2", "password": "Xs-8Q~uaKQaqhGdggDyLmTkwjIa3lNzh2B9AKboR", "tenant": "499a1903-3119-470b-9dd4-d4f436586f25"}'
```

Comment:
We're working remove Azure from openpilot's CI since it's extra infra that only we have access to (you can't fork or make a PR to an azure bucket).

Let's use https://github.com/commaai/ci-artifacts like openpilot does

Reply:
https://github.com/commaai/openpilot/blob/d00a53983089be6bed0e037474abfea823f2fc3e/.github/workflows/ui_preview.yaml#L41

Reply:
yes, `CI_ARTIFACTS_DEPLOY_KEY`. I'm ok trading off some speed for externalized infrastructure.